src/WebJobs.Script/OutOfProc/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -8,5 +8,7 @@ public class HttpWorkerOptions
         public HttpWorkerDescription Description { get; set; }
 
         public WorkerProcessArguments Arguments { get; set; }
+
+        public int Port { get; set; }
```

Comment:
Ah, so the options type is already named HttpWorker - can we then change the host.json config name from httpInvoker to httpWorker to match?